src/styles/layout/header.scss

```
@@ -0,0 +1,126 @@
+.header {
+  box-sizing: border-box;
+  padding: 25px 20px;
+  height: 100vh;
+  background-image: url('/src/images/bg-img.png');
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Good practice divide you css rules by empty line for logic blocks. Cool example here: https://9elements.com/css-rule-order/
```suggestion
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  
  box-sizing: border-box;
  padding: 25px 20px;
  height: 100vh;
  
  background-image: url('/src/images/bg-img.png');
  background-size: cover;
  background-position: center;
```

Don't change codestyle here, but use it in future projects